Lib/test/libregrtest/utils.py

```diff
@@ -54,3 +58,94 @@ def printlist(x, width=70, indent=4, file=None):
     print(textwrap.fill(' '.join(str(elt) for elt in sorted(x)), width,
                         initial_indent=blanks, subsequent_indent=blanks),
           file=file)
+
+BUFSIZE = 8192
```

Comment:
Please put the new code in a new file, like libregrtest/winloadavg.py.

Reply:
Done